src/routeObject.js

```diff
@@ -1,6 +1,51 @@
 import React from 'react';
 import {Route, IndexRoute} from 'react-router';
+
 import App from './pages/App';
+import NotFound from './pages/PageNotFound';
+import store from './store';
+
+const validPathTest = (course, lesson, path) => {
```

Comment:
`validPathTest` bør være så kjapp som mulig, så jeg foreslår at du flytter all utregning som bare trengs gjøres én gang, utenfor metoden, inkl def. av lessons og courses (og å gjøre slice og replace i courses).

Eller enda bedre; hva med å bare lage to arrays med gyldige paths på forhånd (utenfor denne funksjonen), én for courses og én for lessons, og så kan du bare sjekke med `lessonPaths.indexOf(pathToCheck)` eller `coursePaths.indexOf(pathToCheck)` ?